com.liferay.poshi.ide.core/plugin.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+    <extension
+        point="org.eclipse.core.contenttype.contentTypes">
+
+        <content-type
+            base-type="org.eclipse.core.runtime.xml"
+            default-charset="UTF-8"
+            file-extensions="function"
+            id="com.liferay.ide.poshi.functionfile"
+            name="%liferay.poshi.function.file.name"
```

Comment:
But the descdription in here directly, don't use the % variables, we don't need them.